pantheon-bundle/frontend/src/app/components/Chrome/Header/User.tsx

```diff
@@ -27,7 +27,7 @@ class User extends Component<IAppState, IState> {
 
     public render() {
         const dropdownItems = [
-            <DropdownItem key="help" href="/pantheon/docs/assemblies/assembly-pantheon-help.html" target="_blank">Help</DropdownItem>,
+            <DropdownItem key="User guide" href="/pantheon/docs/assemblies/assembly-pantheon-help.html" target="_blank">Help</DropdownItem>,
```

Comment:
```suggestion
            <DropdownItem key="userguide" href="/pantheon/docs/assemblies/assembly-pantheon-help.html" target="_blank">User Guide</DropdownItem>,
```
@theashiot Close! But the `key` attribute is used internally by React and is not user-facing. Given that, renaming it doesn't hurt, and is probably good practice to make it match closely with its intention (although I did remove the space for safety's sake given that this is used by the framework, but it probably didn't make a difference). The human-facing label that I think you're going after is the text contained within the tag itself, which comes later in this line.

I suggested a change; let me know if that aligns with your goal. Thanks!